packages/common/src/common.ts

```diff
@@ -636,67 +702,45 @@ export class Common extends EventEmitter {
     return BigInt(ttd)
   }
 
```

Comment:
I don't think we can remove this function as this is part of the public API without this being a breaking change.  What's driving removal of this?

Reply:
we don't use it at all, let me know if this needs restoration

Reply:
It's not whether we use it or not. It's part of the public API so any downstream consumer of our libraries could use it and this would break their usage. We can't remove functions or change their parameters without it being a breaking change. I'm not opposed it it in principle but we have to be really careful about breaking releases.